src/fillTank.test.js

```diff
@@ -1,11 +1,167 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it(`should changer next parametres 'customer.money'`
+  + `& 'customer.vehicle.fuelRemains'`
+  + `when 'fuelPrice' & 'amount' are given`, () => {
+    const customer = {
+      money: 3000, // остаток денег на счету клиента
```

Comment:
Please, translate comments into English or remove them at all

Reply:
ah. ok. sorry. i forget about it